tests/backends/tests.py

```diff
@@ -55,6 +55,11 @@ def test_initialization_class_attributes(self):
             instance_attr_value = getattr(conn, instance_attr_name)
             self.assertIsInstance(instance_attr_value, class_attr_value)
 
+    @unittest.skipUnless(connection.vendor != 'unknown', 'Test only if vendor is specified')
+    def test_initialization_display_name(self):
+        self.assertIsNotNone(connection.display_name)
```

Comment:
You might assert `BaseDatabaseWrapper.display_name == 'unknown'`, that would make the `assertNotEqual` more meaningful.